src/components/Chat.tsx

```diff
@@ -1,50 +1,77 @@
 'use client';
 
-import { useState, useRef, useEffect } from 'react';
+import { useState, useRef, useEffect, useCallback } from 'react';
+import { Montserrat } from 'next/font/google';
+
+const montserrat = Montserrat({
```

Comment:
pls include it only at the layout level and keep the font same accross the application.